packages/main/src/plugin/kubernetes-context-state.ts

```diff
@@ -193,8 +211,12 @@ export class ContextsManager {
     return this.createInformer<V1Pod>(kc, context, path, listFn, {
       timer: this.podTimer,
       backoff: new Backoff(1000, 60_000, 300),
-      onAdd: _obj => this.setStateAndDispatch(context.name, state => state.podsCount++),
-      onDelete: _obj => this.setStateAndDispatch(context.name, state => state.podsCount--),
+      onAdd: obj => this.setStateAndDispatch(context.name, state => state.resources.pods.push(obj)),
```

Comment:
We will also need to listen on `update` events, go get updated values of resources